plugins/catalog-import/src/components/StepPrepareCreatePullRequest/StepPrepareCreatePullRequest.test.tsx

```diff
@@ -14,23 +14,25 @@
  * limitations under the License.
  */
 
+import { ApiProvider, ApiRegistry } from '@backstage/core-app-api';
 import { catalogApiRef } from '@backstage/plugin-catalog-react';
 import { TextField } from '@material-ui/core';
 import { act, render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import React from 'react';
+import { errorApiRef } from '../../../../../packages/core-plugin-api/src';
```

Comment:
oh no!

Reply:
Lint would be pretty cool here 😆 Sometimes vscode likes to produce these direct imports…

Reply:
https://github.com/backstage/backstage/blob/fa1339e227a558f126879f0693101d68c3e5a2c2/packages/cli/config/eslint.js?plain=1#L86

https://github.com/backstage/backstage/blob/fa1339e227a558f126879f0693101d68c3e5a2c2/packages/cli/config/eslint.backend.js?plain=1#L65

but they don't capture all cases :/ improvements are welcome